docs/content/preview/explore/linear-scalability/scaling-transactions.md

```diff
@@ -31,100 +31,41 @@ type: docs
 -->
 </ul>
 
-This page demonstrates YugabyteDB's horizontal scale-out and scale-in capability. With YugabyteDB, you can add nodes to upscale your cluster efficiently and reliably to achieve more read and write IOPS (input/output operations per second). This tutorial shows how YugabyteDB can scale while running a read-write workload. Using the prepackaged [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps) against a 3-node local cluster with a replication factor of 3, you add nodes while the workload is running. Next, you can observe how the cluster scales out by verifying that the number of read and write IOPS are evenly distributed across all the nodes at all times.
+This page demonstrates YugabyteDB's horizontal scale-out and scale-in capability. With YugabyteDB, you can add nodes to upscale your cluster efficiently and reliably to achieve more read and write IOPS (input/output operations per second). This tutorial shows how YugabyteDB can scale while running a read-write workload. Using the prepackaged [YB Simulation Base Demo application](https://github.com/yugabyte/yb-simulation-base-demo-app) against a 3-node local cluster with a replication factor of 3, you add nodes while the workload is running. Next, you can observe how the cluster scales out by verifying that the number of read and write IOPS are evenly distributed across all the nodes at all times.
 
-This tutorial uses the [yugabyted](../../../reference/configuration/yugabyted/) cluster management utility.
+{{% explore-setup-multi %}}
 
-## Create cluster
+## Observe IOPS per node
 
-Start a new three-node cluster with a replication factor (RF) of `3` and set the number of [shards](../../../architecture/docdb-sharding/sharding/) (also called tablets) per table per YB-TServer to `4` so that you can better observe the load balancing during scale-up and scale-down.
+To view a table of per-node statistics for the cluster, navigate to the [tablet-servers](http://127.0.0.1:7000/tablet-servers) page. The following illustration shows the total read and write IOPS per node. Note that both the reads and the writes are roughly the same across all the nodes, indicating uniform load across the nodes.
 
-Create the first node:
+![Read and write IOPS with 3 nodes](/images/ce/transactions_observe1.png)
 
-```sh
-$ ./bin/yugabyted start \
-                  --base_dir=/tmp/ybd1 \
-                  --listen=127.0.0.1 \
-                  --master_flags "ysql_num_shards_per_tserver=4" \
-                  --tserver_flags "ysql_num_shards_per_tserver=4,follower_unavailable_considered_failed_sec=30"
-```
+![Latency and throughput with 3 nodes](/images/ce/simulation-graph.png)
```

Comment:
To view the latency and throughput on the cluster while the workload is running, navigate to the [simulation application UI](http://127.0.0.1:8000).